examples/android/.gitignore

```diff
@@ -1,8 +1,12 @@
 local.properties
 .idea
 .gradle
+gradle
+gradlew
+gradlew.bat
```

Comment:
I don't see any files matching these patterns in the repo. Are they generated as part of the Gradle build?

Reply:
Those were generated when I imported and built the project with Android Studio.

After thinking about it, those probably need to be checked into git instead of ignored, as they provide the gradle version that was selected when building the project. It is often a good idea to use such a gradlew file instead of running the native gradle from the OS because it guarantees that the used version is compatible with the project.